src/OrchardCore.Modules/OrchardCore.Deployment/Controllers/DeploymentPlanController.cs

```diff
@@ -105,9 +108,49 @@ public async Task<IActionResult> Index(DeploymentPlanIndexOptions options, Pager
                 Pager = pagerShape
             };
 
+            model.Options.DeploymentPlansBulkAction = new List<SelectListItem>() {
+                new SelectListItem() { Text = S["Delete"], Value = nameof(DeploymentPlansBulkAction.Delete) }
+            };
+
             return View(model);
         }
 
+        [HttpPost, ActionName("Index")]
```

Comment:
Also `"Index"` in attributes can be replaced too

Reply:
ok